kurtosis-module/impl/services/contract_helper.ts

```diff
@@ -124,7 +124,7 @@ export async function addContractHelperService(
         "-c",
         // We need to override the CMD because the Dockerfile (https://github.com/near/near-contract-helper/blob/master/Dockerfile.app)
         // loads hardcoded environment variables that we don't want
-        "sleep 10 && yarn start-no-env",
+        "sleep 10 && /usr/dynamodb/local_dynamo.sh && sleep 10 && yarn start-no-env",
```

Comment:
This hack is coupled with https://github.com/near/near-contract-helper/pull/652 onNear Contract Helper to be able to spin up a dynamodb locally (because the service will expect dynamodb to run on localhost...)
TODO: decide what do do with this hack following this discussion: https://github.com/near/near-contract-helper/issues/621#issuecomment-1283779840

Reply:
Not needed anymore when https://github.com/near/near-contract-helper/pull/653 merges